package.json

```diff
@@ -16,5 +16,10 @@
     "client": "cd packages/client && yarn",
     "service": "cd packages/service && yarn",
     "format": "prettier -w packages/*/src/** packages/*/package.json tsconfig.json"
+  },
+  "dependencies": {
+    "pg": "^8.6.0",
```

Comment:
Is the pg dependency necessary?

Reply:
It is necessary for pg-pool, which is used to run the queries through zapatos.